xml/System.Net.Http/HttpVersionPolicy.xml

```diff
@@ -74,7 +92,16 @@
       </ReturnValue>
       <MemberValue>0</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Default behavior, either uses requested version or downgrades to a lower one.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+If the server supports the requested version, either negotiated via ALPN (H2) or advertised via Alt-Svc (H3), as well as a secure connection is being requested, the result is the <see cref="HttpRequestMessage.Version" />. Otherwise, downgrades to HTTP/1.1.
```

Comment:
```suggestion
If the server supports the requested version, either negotiated via ALPN (H2) or advertised via Alt-Svc (H3), and a secure connection is being requested, the result is the <see cref="HttpRequestMessage.Version" />. Otherwise, the version downgrades to HTTP/1.1.
```